<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cooler</class>
 <widget class="QWidget" name="cooler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytClr">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytClr
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytClr.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cmmndBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>153</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cmmndBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/cmmndBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sttsBtn">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>10</y>
     <width>153</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sttsBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/sttsBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbnBtn">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>10</y>
     <width>154</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbnBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mbstBtn">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>10</y>
     <width>155</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mbstBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvlingBtn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>375</y>
     <width>153</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lvlingBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tempClrBtn">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>375</y>
     <width>153</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tempClrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/tempClrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pwrBtn">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>375</y>
     <width>153</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pwrBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/pwrBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckBtn">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>375</y>
     <width>153</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="kab1Btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab1Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/offBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="kab2Btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab2Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/offBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kab1TmpLbl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab1TmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;KABINE &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt; Temp = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="kab2TmpLbl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab2TmpLbl
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;KABINE &lt;/span&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; Temp = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="kab1TmpVal">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab1TmpVal
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="kab2TmpVal">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>240</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#kab2TmpVal
{
color:rgb(56, 54, 55)
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
